### PR TITLE
feat: add skeleton for guardian apply run command

### PR DESCRIPTION
**Pull request by gjonathanhong**

### CODE REVIEW

File: pkg/commands/apply/init.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+// InitCommand is a subcommand of apply and implements the cli.Command interface.
+// It provides the working directories to run terraform apply on.
+type InitCommand struct {
+	cli.BaseCommand
+}
+
+// Desc provides a short, one-line description of the command.
+func (c *InitCommand) Desc() string {
+	return "Determine the working directories to run terraform apply on."
```
Comment (verbanicm) [inline pkg/commands/apply/init.go:32]: Generalize this to and remove period

`Initialize Guardian for running the Terraform apply process`

https://github.com/abcxyz/readability/wiki/Go-Best-Practices#descriptions

File: pkg/commands/apply/init.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+// Help is the long-form help output to include usage instructions and flag
+// information.
+func (c *InitCommand) Help() string {
+	return `
+Usage: {{ COMMAND }} [options] <dir>
+
+	Determine the working directories to run terraform apply on.
```
Comment (verbanicm) [inline pkg/commands/apply/init.go:41]: See comment above, this can contain the period.

File: pkg/commands/apply/init.go [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+// Desc provides a short, one-line description of the command.
+func (c *InitCommand) Desc() string {
+	return "Determine the working directories to run terraform apply on."
+}
+
+// Help is the long-form help output to include usage instructions and flag
+// information.
+func (c *InitCommand) Help() string {
+	return `
+Usage: {{ COMMAND }} [options] <dir>
```
Comment (verbanicm) [inline pkg/commands/apply/init.go:39]: Nit: <directory>

Comment (verbanicm) [reply pkg/commands/apply/init.go:39]: My phone didn't render that appropriately.

**Review by verbanicm — APPROVED** [review]

**Comment by verbanicm** [comment]
nit: rename pr title for run command

**Review by drevell — APPROVED** [review]

**Review by yolocs — APPROVED** [review]

**Review by sethvargo — APPROVED** [review]

**Pull request merged**